ios/REANodesManager.mm

```diff
@@ -187,6 +187,8 @@ - (nonnull instancetype)initWithModule:(REAModule *)reanimatedModule
     _operationsInBatch = [NSMutableDictionary new];
     _componentUpdateBuffer = [NSMutableDictionary new];
     _viewRegistry = [_uiManager valueForKey:@"_viewRegistry"];
+    _eventHandler = ^(id<RCTEvent> event) {
+    };
```

Comment:
Let's mark this block as no-op explicitly:
```suggestion
    _eventHandler = ^(id<RCTEvent> event) {
      // no-op
    };
```
If this block isn't meant to be called, we can add something like `assert(false);` inside to make sure we notice it once it starts to fail.

Reply:
Theoretically there is a very rare case when it could be called, it someone had emitted event before the `registerEventHandler` method was called.